web/src/routes/favorites/favorites.bloc.ts

```diff
@@ -5,19 +5,13 @@ import {
 import { AssetResponseDto, api } from '@api';
 import { writable } from 'svelte/store';
 
-type AssestsProps = { favorites: AssetResponseDto[] };
+export const useFavorites = () => {
+	const favorites = writable<AssetResponseDto[]>([]);
 
-export const useFavorites = (props: AssestsProps) => {
-	const favorites = writable([...props.favorites]);
-
-	async function loadFavorites(): Promise<void> {
+	const loadFavorites = async (): Promise<void> => {
```

Comment:
This can be merged into the favorites page, and we probably don't need a store, since it'll only be used on that single page. Something like this:

```typescript
let favorites: AssetResponseDto[] = [];

onMount(async() => {
  try {
    const { data: assets } = await api.assetApi.getAllAssets(true);
    favorites = assets;
  catch(error) {
    handleError(error, 'Unable to load favorites');
  }   
});
```